### ISSUE TITLE
SDK have errors while creating LAML bins 

### ISSUE THREAD

**Issue by ShashiKumar-SignalWire** [signalwire/signalwire-python-old#8]
`from signalwire.voice_response import VoiceResponse, Dial, Number`
 for the above line I am getting the error

```Python 2.7.13 (default, Sep 26 2018, 18:42:22)
[GCC 6.3.0 20170516] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from signalwire.voice_response import VoiceResponse, Dial, Number
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name Dial
```

please check

**Comment by ikcam** [comment]
You can use `from twilio.twiml.voice_response import Dial, Number` as it gets updated.

**Comment by lpradovera** [comment]
Fixed by #10
